prod/westeurope/support/functions_backoffice/function_app/terragrunt.hcl

```diff
@@ -109,6 +113,11 @@ inputs = {
       ADB2C_CLIENT_KEY = "bo-CLIENT-KEY"
       ADB2C_TENANT_ID  = "bo-TENANT-ID"
 
+      // REDIS
+      REDIS_URL      = dependency.redis.outputs.hostname
+      REDIS_PORT     = dependency.redis.outputs.ssl_port
+      REDIS_PASSWORD = dependency.redis.outputs.primary_access_key
+
```

Comment:
We should add even ```NODE_ENV                     = "production"``` used in [here](https://github.com/pagopa/io-functions-backoffice/pull/45/files#diff-8f4625a1cf4cb245a2f9e780a9a17ff6a7132b8e070997144a67499d7ff793b4R101)

Reply:
It's already set this [env](https://github.com/pagopa/io-infrastructure-live-new/blob/049e03408f7b8e06f459f2d154fe19b2ac11100a/prod/westeurope/support/functions_backoffice/function_app/terragrunt.hcl#L70)

Reply:
Thanks, sorry I've missed it.